CorsixTH/Lua/hospital.lua

```diff
@@ -2365,6 +2367,13 @@ function Hospital:getRandomBusyRoom()
   if #chosen_room.door.queue >= busy_threshold then return chosen_room end
 end
 
+--! Reset the death counts (key losing conditions)
+function Hospital:resetDeathCount()
+  self.num_deaths = 0
+  self.num_deaths_this_year = 0
+  self.statistics.deaths = 0
+end
```

Comment:
Good that you moved that to here.

Function name is perhaps a bit too neutral, hospitals normally don't reset death counts, so it's not a regular function you'd use in normal hospital computations.
Not sure if you want to fix that, adding some comment may also do the trick?